### DISCUSSION TITLE
Unique identifier

### DISCUSSION THREAD

**Discussion by ssenthu** [openthread/openthread#9632] [Q&A] [answered]
Could you please confirm if the unique identifier of a Thread network that we can rely on is the Extended PAN ID? I assume it is.

I also assume that two adjacent Thread networks can have the same PANID, NAME, NETWORKKEY, etc., but they must not have the same Extended PAN ID. Is that correct?
@jwhui

**Comment by jwhui** [comment, marked as answer]
The Thread Specification defines Extended PAN ID as:
> A random identifier for a Personal Area Network (PAN) that uniquely identifies a Thread Network.

However, Extended PAN ID is not enough to keep devices in their own network. If they share the same Network Key, they will attempt to merge together.

**Reply by ssenthu** [reply]
Thanks @jwhui . Can we assume, unique extend panid and networkkey will keep two thread networks separately even other paramters like panid, networkname, channels are same?

**Reply by jwhui** [reply]
As long as Network Key is different, devices will not form a Thread connection.

**Reply by ssenthu** [reply]
Thanks a lot @jwhui

**Comment by ssenthu** [comment]
answered
